plugins/modules/template_intent.py

```diff
@@ -265,282 +525,575 @@
+                                type: str
+                              selectionValues:
+                                description: Selection values.
+                                type: dict
+                            type: dict
+                        type: list
+                      tags:
+                        description: Configuration Template Import Template's tags.
+                        elements: dict
+                        suboptions:
+                          id:
+                            description: UUID of tag.
+                            type: str
+                          name:
+                            description: Name of tag.
+                            type: str
+                        type: list
+                      template_content:
+                        description: Template content.
+                        type: str
+                      templateParams:
+                        description: Configuration Template Import Template's templateParams.
+                        elements: dict
+                        suboptions:
+                          binding:
+                            description: Bind to source.
+                            type: str
+                          customOrder:
+                            description: CustomOrder of template param.
+                            type: int
+                          dataType:
+                            description: Datatype of template param.
+                            type: str
+                          defaultValue:
+                            description: Default value of template param.
+                            type: str
+                          description:
+                            description: Description of template param.
+                            type: str
+                          displayName:
+                            description: Display name of param.
+                            type: str
+                          group:
+                            description: Group.
+                            type: str
+                          id:
+                            description: UUID of template param.
+                            type: str
+                          instructionText:
+                            description: Instruction text for param.
+                            type: str
+                          key:
+                            description: Key.
+                            type: str
+                          notParam:
+                            description: Is it not a variable.
+                            type: bool
+                          order:
+                            description: Order of template param.
+                            type: int
+                          paramArray:
+                            description: Is it an array.
+                            type: bool
+                          parameterName:
```

Comment:
are we still using camelcase?

Reply:
Export and import template and projects are blocked so we are using camel case similar to one to one API.